src/llmcompressor/entrypoints/train.py

```diff
@@ -9,7 +11,7 @@
 from .utils import post_process, pre_process
 
 
-def train(**kwargs):
+def train(**kwargs) -> PreTrainedModel:
```

Comment:
train returns a model but `oneshot` does not, right? it modifies in place? wondering if we should be consistent across the top-level functions

Reply:
Never mind, looks like `oneshot` also returns a model, based on your code elsewhere in this PR

Reply:
marking as resolved, i misunderstood the `oneshot` function signature